adam/__init__.py

```diff
@@ -0,0 +1,2 @@
+from adam.Computations import KinDynComputations
+from adam.Geometry import utils
```

Comment:
Python module names [are typically lower-case](https://www.python.org/dev/peps/pep-0008/#package-and-module-names). Then, try to avoid importing lower-level classes to higher level packages. This is handy for refactoring and deprecating, but since this is a new library let's avoid polluting higher-level packages with imported classes. 

```suggestion
from . import computations
from . import geometry
```

Reply:
Thanks for this :D . Done in 419d16e